cypress/integration/ContributionForm_spec.js

```diff
@@ -1,31 +1,54 @@
-describe('ContributionForm', () => {
-  beforeEach(() => {
-    cy.visit('http://localhost:3000/submit')
-  })
+describe('ContributionForm View', () => {
+
+    beforeEach(() => {
+      cy.fixture('mockDr.json')
+        .then(mockData => {
+          cy.intercept('GET', 'https://head-to-toe-be.herokuapp.com/api/v1/medical_professionals?type=doctor&state=South Dakota', {
+            statusCode: 201,
+            delay: 100,
+            body: mockData
```

Comment:
I don't know if we need to intercept a Get request here, since we're not displaying cards on this page